src/analysis/retail/druid/restoration/modules/spells/Dreamstate.tsx

```diff
@@ -0,0 +1,61 @@
+import { TALENTS_DRUID } from 'common/TALENTS';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import Events, { CastEvent } from 'parser/core/Events';
+import Statistic from 'parser/ui/Statistic';
+import { SpellIcon } from 'interface';
+import BoringValue from 'parser/ui/BoringValueText';
+import { formatNumber } from 'common/format';
+
+const cdrPerTick = 3;
```

Comment:
All times and durations in both log and WoWA are in milliseconds, so this should be 3000.

The one exception is cooldowns as defined in the Abilities file, and I know this is kinda close to that but `SpellUsable` is going to want milliseconds so I'd prefer this was in millis too.